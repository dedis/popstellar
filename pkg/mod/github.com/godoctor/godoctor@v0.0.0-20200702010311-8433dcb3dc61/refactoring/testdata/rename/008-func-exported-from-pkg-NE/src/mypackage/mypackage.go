package mypackage

func MyFunction(n int) int { // <<<<< rename,3,6,3,6,xyz,pass
	if n == 0 {
		return 1
	} else {
		return n + MyFunction(n-1)
	}
}
