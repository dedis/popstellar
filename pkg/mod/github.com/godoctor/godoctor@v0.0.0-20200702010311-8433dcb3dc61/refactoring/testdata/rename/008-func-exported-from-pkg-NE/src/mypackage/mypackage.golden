package mypackage

func xyz(n int) int { // <<<<< rename,3,6,3,6,xyz,pass
	if n == 0 {
		return 1
	} else {
		return n + xyz(n-1)
	}
}
